src/components/pricedButton/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Button from 'react-toolbox/lib/button';
+import { fromRawLsk } from '../../utils/lsk';
+import styles from './pricedButton.css';
+
+const PricedButton = (props) => {
+  const hasFunds = props.balance >= props.fee;
+  return (
+    <div className='primary-button'>
+      {
+        props.fee &&
+          <span className={`${styles.fee} ${hasFunds ? '' : styles.error}`}>
```

Comment:
Maybe it is a better solution for changing `classnames` based on state https://github.com/JedWatson/classnames